### PR TITLE
Added localhost:4000/uframe/streams_list . Returns JSON object.  Json ke...

### PR DESCRIPTION
**Pull request by DanielJMaher**
...ys are streams.  inside each stream is the start time, end time, refdes, and download link

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@oceanzus @lukecampbell Please Review

**Comment by oceanzus** [comment]
Umm, travis says it is bad news....
ImportError: No module named flask.ext.cache

**Comment by DanielJMaher** [comment]
@oceanzus Odd....

**Comment by lukecampbell** [comment]
We'll need to add 'Flask-Cache' to the requirements.

**Comment by lukecampbell** [comment]
It's a requirement in the UI so that's why your virtualenv has it probably. But the services needs it too. Could you update requirements/common.txt to include it
make sure it's the same version as the one in the UI.

**Comment by DanielJMaher** [comment]
@oceanzus @lukecampbell I added the reqs, but good thing it didn't build properly.  I found a bug that wasn't showing up in my notebook, but was occurring on the page.  On a second inspection i caught it.  The fix is in this pull req too.

**Comment by DanielJMaher** [comment]
It says it can't find the config.... but it is there, isnt it?

**Comment by DanielJMaher** [comment]
@lukecampbell @oceanzus ALL IS WELL!

File: ooiservices/config/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+TABLEDAP: http://data-test.ooi.rutgers.edu/erddap/tabledap/
+ERDDAP_URL: http://data-test.ooi.rutgers.edu/erddap
+DEBUG: False
+UFRAME_DATA: http://localhost:12570/sensor/m2m/inv
+CACHE_TYPE: simple
```
Comment (lukecampbell) [inline ooiservices/config/config.yml]: what's this file for?

File: ooiservices/app/uframe/controller.py [hunk 2]
```diff
@@ -17,15 +18,22 @@
...
 
+UFRAME_DATA = 'http://localhost:12570/sensor/m2m/inv'
+
 #ignore list for data fields
 FIELDS_IGNORE = ["stream_name","quality_flag"]
 #time minus ()
 COSMO_CONSTANT = 2208988800
 
+app = Flask(__name__)
+cache = Cache(app, config={'CACHE_TYPE': 'simple'})
```
Comment (lukecampbell) [inline ooiservices/app/uframe/controller.py]: these should be in the application config file

File: ooiservices/config/config.yml (hunk 1 above)
Comment (DanielJMaher) [inline ooiservices/config/config.yml]: This was originally used for the UFRAME_DATA variable and I was going to use it for the CACHE_TYPE as well.  Straight Copied from front end.

**Pull request merged**